Jenkinsfile_nightly

```diff
@@ -59,27 +59,20 @@ withNightlyPipeline("nodejs", product, component) {
     env.TEST_E2E_WAIT_FOR_ANGULAR = 'false'
     env.TEST_E2E_FAIL_FAST = 'false'
     env.TEST_E2E_ANNOTATION = params.NIGHTLY_TAG
-    enableFullFunctionalTest(120)
+    //enableFullFunctionalTest(120)
```

Comment:
Any reasons why we commented this line, This will enable test and add more time to pipeline(2hrs)

Reply:
> Any reasons why we commented this line, This will enable test with @nightyly-tag and add more time to pipeline(2hrs)

This was commented to speed up our test process. Now it has been reverted as part of ticket  https://github.com/hmcts/ia-ccd-e2e-tests/pull/409/files since this branch has been merged)